macroid-core/src/main/scala/macroid/contrib/ExtraTweaks.scala

```diff
@@ -20,9 +20,10 @@ object TextTweaks {
   val italic = Tweak[W](x ⇒ x.setTypeface(x.getTypeface, Typeface.ITALIC))
   val boldItalic = Tweak[W](x ⇒ x.setTypeface(x.getTypeface, Typeface.BOLD_ITALIC))
 
-  val serif = Tweak[W](x ⇒ x.setTypeface(Typeface.SERIF, Option(x.getTypeface).map(_.getStyle).getOrElse(Typeface.NORMAL)))
-  val sans = Tweak[W](x ⇒ x.setTypeface(Typeface.SANS_SERIF, Option(x.getTypeface).map(_.getStyle).getOrElse(Typeface.NORMAL)))
-  val mono = Tweak[W](x ⇒ x.setTypeface(Typeface.MONOSPACE, Option(x.getTypeface).map(_.getStyle).getOrElse(Typeface.NORMAL)))
+  val serif = Tweak[W](x ⇒ x.setTypeface(Typeface.SERIF, styleOf(x)))
+  val sans = Tweak[W](x ⇒ x.setTypeface(Typeface.SANS_SERIF, styleOf(x)))
+  val mono = Tweak[W](x ⇒ x.setTypeface(Typeface.MONOSPACE, styleOf(x)))
+  def font(name: String) = Tweak[W](x ⇒ x.setTypeface(Typeface.create(name, 0), styleOf(x)))

```

Comment:
I think calling it `typeface` would be more logical, as you are not supplying an actual font file. Also I know this package has no scaladocs, but could you add one here with an example? The parameter, being a string, might not be obvious.